src/core/plugins/deep-linking/helpers.js

```diff
@@ -1,7 +1,40 @@
-export const setHash = (value) => {
+const setHash = (value) => {
   if(value) {
     return history.pushState(null, null, `#${value}`)
   } else {
     return window.location.hash = ""
   }
 }
+
+
+
+const makeDeeplinks = (rootNode) => {
+  /**
+    Converts an element's qualified descendants into deep links. A qualified
+    descendant is an anchor element whose href attribute is assigned the hash of an
+    operation or tag, and whose name attribute is assigned `deeplink`. A qualified
+    descendant becomes a deep link when it can dispatch a deeplinkClick event whenever
+    the descendant is clicked.
+
+    Arguments:
+      a) rootNode: element object
+
+**/
+  const callback = function() {
+    const deeplinks = rootNode.querySelectorAll("[name=deeplink]")
```

Comment:
Swagger-UI should not select elements from across the document, or use APIs that expose the entire DOM. Concerns here include:
- collisions with user applications: I'm integrating Swagger-UI into an application that has `<a name="deeplink">` elements in it for whatever reason... what breaks?
- collisions with other Swagger-UI instances: what happens if Swagger-UI is loaded twice on the page? For obvious reasons, we can't support two deeplinking instances, but if one of the instances have it enabled, deep linking content in the other instance should not be detected by the one with the feature enabled.

---

A better approach would be to modify the Markdown component: currently, it renders all links with `target="_blank"` so that they all open in a new tab. If that is disabled for links that begin with `#`, one can trigger changes to the URL fragment from a standard Markdown link `[my link](#/tag/operation)`.